### PR TITLE
SS-508 removing scaleout stuff

### PR DESCRIPTION
**Pull request by churnikov**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description

Mostly removed usage of scaleout images

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [ ] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [ ] I have updated the release notes (releasenotes.md)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [ ] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

**Review by akochari — APPROVED** [review]
Looks good to me! but the tests didn't run for some reason?

File: templates/admin/base_site.html [hunk 1]
```diff
@@ -7,11 +7,11 @@
     
 {% endblock %}
 {% block branding %}
-    <img src="{% static 'images/scaleout_black.png' %}" alt="Studio Logo" width="200">
+    <img src="{% static 'images/serve_logo.svg' %}" alt="Studio Logo" width="200" height="30px">
```
Comment (sandstromviktor) [inline templates/admin/base_site.html:10]: Perhaps change the `alt` aswell?

File: templates/registration/signup.html [hunk 2]
```diff
@@ -14,7 +14,9 @@
                     <div class="px-3 py-5">
                         <div class="row">
                             <div class="col col-lg-6 offset-lg-3">
-                                <img src="{% static 'images/scaleout_black.png' %}" alt="Studio Logo" style="max-width: 100%; height: auto;">
```
Comment (sandstromviktor) [inline templates/registration/signup.html:17]: It feels a bit empty, but perhaps we can add some logo later?

**Review by sandstromviktor — APPROVED** [review]
Great!

File: templates/registration/signup.html (hunk 2 above)
Comment (churnikov) [reply templates/registration/signup.html:17]: I thought of adding logos as it was done by scaleout, but in my opinion, there would be too much visual elements with additional logo

**Pull request merged**